packages/react-stream/src/ReactFizzHostConfigNode.js

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
@gaearon I put this in `react-stream/src/` which is a bit weird because it's not part of the core algorithm. It's part of a possible host config. I'm not sure this makes sense to put it here but it's not really `shared` and we really shouldn't make `shared` a dumping ground.

However, I guess in the future we could have `react-stream` export preconfigured environments and all you fill in is the format config. E.g. `react-stream/index.node.js`, `react-stream/index.browser.js`.

So maybe that makes sense.

Reply:
Sure. We also never got these things quite right from first try so I wouldn’t expect to do it now.

Reply:
Maybe you can add a comment about this? I was just puzzled over this because I forgot.